var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -26,7 +26,10 @@ class Legion(CMakePackage):
     git      = "https://github.com/StanfordLegion/legion.git"
 
     version('develop', branch='master')
```

Comment:
You can rename this to master now that #1983 has been merged.

Reply:
Ok!